models/spleen_deepedit_annotation/configs/evaluate.json

```diff
@@ -0,0 +1,183 @@
+{
+    "imports": [
+        "$import glob",
+        "$import os",
+        "$import ignite"
+    ],
+    "bundle_root": "/workspace/tutorials/deepedit/deepedit_bundle",
+    "output_dir": "$@bundle_root + '/eval'",
+    "dataset_dir": "/workspace/Datasets/MSD_datasets/Task09_Spleen",
+    "datalist": "$list(sorted(glob.glob(@dataset_dir + '/imagesTs/*.nii.gz')))",
```

Comment:
Thanks @diazandr3s for the quick updates. This json file seems the same as `inference.json`, and does not have any metric related settings. Here we need a file that can produce a validation score that is exactly the same as the readme file, thanks!

Reply:
Hi @diazandr3s ,

Please refer to this one: https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/evaluate.json

Thanks.

Reply:
Thanks for your comments, @yiheng-wang-nv @Nic-Ma 
I've just updated the PR according to your feedback